_posts/2019-02-01-dynamic-spawn-scheduler.md

```diff
@@ -23,17 +23,23 @@ Action mnemonics are statically mapped to strategies via command-line flags. Whe
 
 ## The problem with remote execution
 
-Unfortunately, in the case of remote execution, sending *all* actions through the `remote` strategy is inefficient. Each remote action comes with a cost: there is the overhead of the RPCs to contact with the remote service and there is also the more-significant overhead of uploading inputs and downloading outputs (even if we can [maybe optimize the latter](https://docs.google.com/document/d/11m5AkWjigMgo9wplqB8zTdDcHoMLEFOSH0MdBNCBYOE/edit)). As a result, blindly running all actions remotely can be (and often is) detrimental to performance.
+In the case of a fully remote build, sending *all* actions through the `remote` strategy can be inefficient. Each remote action comes with a cost: there is the overhead of the RPCs to contact with the remote service and there is also the more-significant overhead of uploading inputs and downloading outputs (even if we can [maybe optimize the latter](https://docs.google.com/document/d/11m5AkWjigMgo9wplqB8zTdDcHoMLEFOSH0MdBNCBYOE/edit)). As a result, blindly running all actions remotely can be (and often is) detrimental to performance.
```

Comment:
Two small writing nits: It should be "more significant" instead of "more-significant". I would also say "even if we may be able to optimize the latter" rather than "can maybe".

Reply:
Done.